### PR TITLE
MA-3591 : Change the Account creation StateMachine to trigger CloudCheckr StateMachine: 

### PR DESCRIPTION
**Pull request by aniruddha-sungardas**
**Purpose**

Modify template to call TriggerCloudChecker after UpdateDb.  This is applicable only for managed account.

**Changes**

Template changes

**Testing**

Tested by creating new state machine & running it.

### CODE REVIEW

**Comment by jophine-antony** [comment]
@bhargavapratik no it is been added after update db step because cloud checker statemachine need not wait until further steps are completed.

@SonalOjha  please confirm

File: template.yaml [hunk 1]
```diff
@@ -549,6 +549,11 @@ Resources:
               "Type": "Task",
               "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-Update-Db",
               "InputPath": "$",
+              "Next": "TriggerCloudCheckr"
+            },
+            "TriggerCloudCheckr": {
+              "Type" : "Task",
+              "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Onboarding-TriggerCloudChecker",
```
[the remark this answers is not in the document]
Comment (jophine-antony) [reply template.yaml]: input and output are defaulted to '$' and are optional

**Comment by SonalOjha** [comment]
@bhargavapratik After update DB the CloudCheckr Lambda is triggerred as the next steps to it are all related to monitoring. So we added the step to invoke cloudcheckr statemachine right after that.

File: template.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (SonalOjha) [reply template.yaml]: Having the entire input sent to the next step is fine for now as then one of the step has to prepare the JSON required by the Lambda. If the entire input is send the Lambda parses and retrieves whats required by it.

Comment (RajkumarShivage) [inline template.yaml]: Spell is cloudcheckr not cloudchecker.

Comment (RajkumarShivage) [inline template.yaml]: Who will deploy this lambda function?

Comment (jophine-antony) [reply template.yaml]: This lambda function will be deployed as part of cloud checker stack which is being worked on by @mayankbharadwaj

[the remark this answers is not in the document]
Comment (bhargavapratik) [reply template.yaml]: We can filter inputs and send it to Lambda.

**Pull request merged**